strawberryfields/gbs/similarity.py

```diff
@@ -345,6 +351,7 @@ def prob_orbit_mc(graph: nx.Graph, orbit: list, n_mean: float = 5, samples: int
         orbit (list[int]): orbit for which to estimate the probability
         n_mean (float): total mean photon number of the GBS device
         samples (int): number of samples used in the Monte Carlo estimation
+        loss (float): fraction of photons lost in GBS
 

```

Comment:
This function probably needs to raise an error if the loss is not between 0 and 1

Reply:
This has been added in: https://github.com/XanaduAI/strawberryfields/pull/223/commits/0035650994a5feb5f643201f54ce9eee4c913885, along with some other tests that were missing. I feel we could probably get away with having fewer tests though.